### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by MeycL** [huoxiangzuo/HiFuse#3]
Thank you very much for your work open source, can you please provide the address of the data? This may be more friendly to newbies.

**Comment by huoxiangzuo** [comment]
Sorry, we are using a public dataset and do not have permission to publish it.

You can download it from the publisher of the dataset.

**Comment by MeycL** [comment]
Thank you for your answer, and I also strongly agree with you. But I don't really know how to preprocess the dataset after getting it, can you please provide me with a way to preprocess the data for the three datasets (e.g. python file .py)? I would be grateful if you could.

**Comment by huoxiangzuo** [comment]
You can use this method to divide the dataset into training set and validation set, attach the link: https://github.com/WZMIAOMIAO/deep-learning-for-image-processing/tree/master/data_set
├── flower_data   
       ├── flower_photos（解压的数据集文件夹，3670个样本）  
       ├── train（生成的训练集，3306个样本）  
       └── val（生成的验证集，364个样本）

**Comment by jiaoxueping** [comment]
您好，可以提供一下ISIC2018数据集分配的代码吗，论文中说训练集和验证集占百分之七十，那训练集和验证集又是如何分配的呢，希望能提供一下分配的代码
